_episodes/40.md

```diff
@@ -0,0 +1,481 @@
+---
+layout: episode
+title: "40: Trino's cold as Iceberg!"
+date: 2022-09-08
+tags: iceberg apache datalake table format sql schema evolution partition optimize
+youtube_id: "6NyfCV8Me0M"
+wistia_id: "b9nlw7rjpr"
+sections: 
+   - title: "Concept of the week"
```

Comment:
Concept of the episode 

also maybe even better

Concept

or 

Topic

since there are so many